include/SFML/System/Clock.hpp

```diff
@@ -30,10 +30,34 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/System/Export.hpp>
 #include <SFML/System/Time.hpp>
+#include <chrono>
+#include <ratio>
```

Comment:
The `<ratio>` header (and the `static_assert`s) could probably be moved to the .cpp file, to reduce header dependencies.

Reply:
`<ratio>` is already part of `<chrono>`. I added that for clarity.

I couldn't do it because `ClockImpl` is a private member of the class so `sf::Clock::ClockImpl` is hidden outside of the class. I could make it work by putting it into the constructor but I didn't want to do that because then every other constructor in the future would have to include that too.

There is one other reason though. I don't know if that's the rule but I think `static_assert`'s should reside in header files when affecting the class as a whole. I was just able to code a mini example that would suppress static assertion when put in the source file.

Reply:
Makes sense 👍